### PR TITLE
 Added some Civilian Structures

### PR DESCRIPTION
**Pull request by MustaphaTR**
Factory, Gas Station, Einstein's Lab and Communications Center.

### CODE REVIEW

**Comment by MustaphaTR** [comment]
Wanted to say before @Mailaender does. Needs rebase, i can see it.

**Comment by MustaphaTR** [comment]
Rebased.

File: rules/civilian-structures.yaml [hunk 1]
```diff
@@ -314,6 +314,7 @@ causfgl:
 	Inherits: ^Flag
 	Tooltip:
 		Name: US Flag
+<<<<<<< 2da4f2a33cecaeb832b11976a35651ab19fed98c
```
Comment (abcdefg30) [inline rules/civilian-structures.yaml]: This doesn't look right.

Comment (Phrohdoh) [inline rules/civilian-structures.yaml]: Merge conflict marker!

**Comment by MustaphaTR** [comment]
Updated. Also removed `Selectable:` from those too.

File: rules/civilian-structures.yaml [hunk 2]
```diff
@@ -358,3 +358,60 @@ caukfgl:
 	Inherits: ^Flag
 	Tooltip:
 		Name: British Flag
+
+caind01:
+	Inherits: ^CivBuilding
+	Selectable:
```
Comment (abcdefg30) [inline rules/civilian-structures.yaml]: This one still has `Selectable`.

**Comment by MustaphaTR** [comment]
Updated.

**Comment by abcdefg30** [comment]
Sorry, needs a rebase again.

**Comment by MustaphaTR** [comment]
Rebased.

**Comment by Phrohdoh** [comment]
Let's get this reviewed and either merged or rejected and not make him rebase anymore.

**Comment by abcdefg30** [comment]
Looks good to me. :+1: / :white_check_mark: 
openra-2016-01-15t183741z

**Pull request merged**